### ISSUE TITLE
Pagination a11y error with axe-core 4.5.0+

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#5036]
Updating `axe-core  >=4.5.0` causes a possible false positive with pagination.

<details>
  <summary>Pagination a11y error</summary>

  ```bash
1) [chromium] accessibility
     Components/Pagination
       Default

       Detected the following accessibility violations!

       1. list (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)

          For more info, visit https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI.

          Check these nodes:

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

2) [chromium] accessibility
     Components/Pagination
       Unbounded

       Detected the following accessibility violations!

       1. list (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)

          For more info, visit https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI.

          Check these nodes:

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

3) [chromium] accessibility
     Components/Pagination
       Spanish

       Detected the following accessibility violations!

       1. list (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)

          For more info, visit https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI.

          Check these nodes:

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

4) [chromium] accessibility
     Components/Pagination
       Spanish Unbounded

       Detected the following accessibility violations!

       1. list (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)

          For more info, visit https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI.

          Check these nodes:

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

5) [chromium] accessibility
     Components/Pagination
       Test

       Detected the following accessibility violations!

       1. list (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)

          For more info, visit https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI.

          Check these nodes:

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]

          - html: <ul class="usa-pagination__list">
            summary: Fix all of the following:
                       List element has direct children that are not allowed: [role=presentation]
  ```
</details>

_Originally posted by @mejiaj in https://github.com/uswds/uswds/issues/5019#issuecomment-1307796280_

**Comment by mejiaj** [comment]
Duplicate, closing in favor of https://github.com/uswds/uswds/issues/5022
